tasks/main.yml

```diff
@@ -98,3 +98,8 @@
 - name: Install from Brewfile.
   command: "brew bundle chdir={{ homebrew_brewfile_dir }}"
   when: homebrew_brewfile.stat.exists and homebrew_use_brewfile
+
+# Services.
+- name: Start services at login.
+  command: "brew services start {{ item }}"
+  with_items: "{{ homebrew_services }}"
```

Comment:
Since there's no condition on this, it wouldn't be idempotent—is there a way we can check for running services and only run the command if a service is not already running?